odoo/addons/base/module/wizard/base_import_language_view.xml

```diff
@@ -8,8 +8,11 @@
             <field name="arch" type="xml">
                 <form string="Import Translation">
                     <group>
-                        <field name="name" placeholder="e.g. English"/>
-                        <field name="code" string="Code" placeholder="e.g. en_US"/>
+                        <field name="lang"/>
+                        <field name="name" placeholder="e.g. English"
+                               attrs="{'invisible': [('lang', '!=', 'new_language')]}"/>
+                        <field name="code" string="Code" placeholder="e.g. en_US"
+                               attrs="{'invisible': [('lang', '!=', 'new_language')]}"/>
                         <field name="data" filename="filename"/>
                         <field name="filename" invisible="1"/>
                         <field name="overwrite"/>
```

Comment:
can hide this one for new languages too

Reply:
"code" is required when creating new language by "load_lang" function in res.lang model.